sdk/cosmosdb/cosmos/api-extractor.json

```diff
@@ -1,24 +1,31 @@
 {
```

Comment:
Updated `"@microsoft/api-extractor"` to version `"^7.1.5"` to match with the other packages and also had to update the api-extractor.json as per the template.